### ISSUE TITLE
Dynamic shadows sometimes show on the wrong side of walls

### ISSUE THREAD

**Issue by henke37** [mastercomfig/team-comtress-2#135]
**Describe the bug**
This is one of the most notorious issues with shadow rendering in the engine.

**How you encountered the issue**
Tends to appear while normally playing.

This is the issue that `info_no_dynamic_shadow` tries to hide. Better fix it for real and scrap the hack to hide the issue.

**Comment by maximsmol** [comment]
This is technically a pretty big balance change lol, thus RFC

**Comment by Shrinks99** [comment]
I don't think this should be viewed as a balance change :P  I've gotten a lot of kills that I really shouldn't have in game due to this bug and it feels like I'm cheating every time.  I don't think people would be sad to see it go.

**Comment by Yttrium-tYcLief** [comment]
Yeah no, this is definitely a bugfix and within scope, but is going to require a lot of effort to fix.
